README.md

```diff
@@ -158,8 +158,17 @@ sudo make install
 
 Sandman has a web interface implemented with Flask. These instructions cover what you need to do in order to run this web interface in development mode. In the future there will be a way to run this web interface in a more production friendly environment.
 
-Installation instructions can be found here: [Flask Installation](https://flask.palletsprojects.com/en/3.0.x/installation/). However, they are not complicated and we can skip a step, so they will be provided here. Assuming that your Sandman repository is stored in the home directory, it is recommended to switch to the Sandman web directory in order to set up the virtual Python environment. This is not strictly required, but is recommended by Flask.
+Installation instructions can be found here: [Flask Installation](https://flask.palletsprojects.com/en/3.0.x/installation/). However, they are not complicated and we can skip a step, so they will be provided in the following instructions.
 
+Install byobu
```

Comment:
I didn't know about byobu before, but I took a look to understand what this was. It seems pretty interesting, but definitely not required. I don't think these instructions really make it clear at all that this is completely optional and as such, I don't think we should be including instructions on it. If we want to plug it, that's fine, but I would just say that you might want to consider using it and provide a link to it for people that are curious. In terms of instructions I think we should focus on the minimum number of required steps except in cases where there is some recommendation by the software we are using.

Reply:
I understand and I'm willing to make it clear that using byobu is optional and remove the instructions related to it.